libs/modules/users/application/src/lib/users-service-factory.ts

```
@@ -0,0 +1,20 @@
+
+export class UsersServiceFactory {
+  private static instance: UsersService | null = null;
+
+  static initialize(repository: UsersRepository): void {
+    if (!this.instance) {
+      this.instance = new UsersService(repository);
+    }
+  }
+
+  static getInstance(): UsersService {
+    if (!this.instance) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)